tests/search/max_document_size/max_document_size.rb

```diff
@@ -0,0 +1,51 @@
+# Copyright Vespa.ai. All rights reserved.
+require 'document'
+require 'indexed_only_search_test'
+
+class MaxDocumentSize < IndexedOnlySearchTest
+
+  def setup
+    set_owner("hmusum")
+    @id = 0
+    @one_mib = 1024 * 1024
+  end
+
+  def test_max_document_size
+    deploy_application('1MiB')
+    start
+    vespa.adminserver.execute("vespa-get-config -n vespa.config.content.core.stor-distributormanager -i mystorage/distributor/0 | grep max_document_operation_message_size_bytes")
+
+    # feed a bit less than 1 MiB
+    feed(@one_mib - 100)
+
+    # feed a bit more than 1 MiB, should fail
+    begin
+      feed(@one_mib + 100)
+      raise "An error should have been raised when feeding more than 1 MiB"
+    rescue HttpResponseError => e
+      # as expected
```

Comment:
Should we test the concrete HTTP response code generated?

Reply:
Yes, that as well as verify the error message. Right now feed of doc > 1MiB does NOT fail, so need to figure out what's wrong with test or config/code.

Reply:
Added